variables.tf

```diff
@@ -225,53 +225,50 @@ variable "gcp_sa_private_key" {
   default     = null
 }
 
-variable "gcp_sa_project_id" {
+# GCP Storage Configuration
+variable "gcp_artifact_registry_url" {
   type        = string
-  description = "GCP Project ID associated with the Service Account"
+  description = "URL for GCP Artifact Registry (e.g., 'LOCATION-docker.pkg.dev/PROJECT_ID/REPOSITORY')"
   default     = null
 }
 
-# GCP OAuth Configuration
-variable "gcp_sa_auth_uri" {
+variable "gcp_storage_bucket_url" {
   type        = string
-  description = "GCP OAuth authentication URI"
-  default     = "https://accounts.google.com/o/oauth2/auth"
+  description = "URL for GCP Storage bucket (e.g., 'gs://bucket-name')"
+  default     = null
 }
 
-variable "gcp_sa_token_uri" {
-  type        = string
-  description = "GCP OAuth token URI"
-  default     = "https://oauth2.googleapis.com/token"
+variable "gcp_cluster_integration_enabled" {
```

Comment:
what does this variable mean?

Reply:
`cluster_integration_enabled` is used if you want to skip integration https://github.com/truefoundry/terraform-truefoundry-integrations/blob/main/templates/aws.json.tpl#L50  
this was something ocli also used to pass, by default its true.